springhello/src/main/java/telran/UserController.java

```diff
@@ -8,8 +8,7 @@
 
 import java.util.List;
 
-import static org.springframework.web.bind.annotation.RequestMethod.GET;
-import static org.springframework.web.bind.annotation.RequestMethod.POST;
+import static org.springframework.web.bind.annotation.RequestMethod.*;
```

Comment:
It is better not to use wildcards in imports but rather explicitly list every imported class.

Reply:
ok thanks